src/commands/extract-translations-and-store-command.ts

```diff
@@ -47,7 +48,7 @@ export class ExtractTranslationsAndStoreCommand implements Command {
 	private async parseTermsFiles(): Promise<ExtractedKey<SnapshotData>[]> {
 		let keys: ExtractedKey<SnapshotData>[] = [];
 		for (const source of this.configuration.terms) {
-			const reader = await ReaderFactory.factory(source.type);
+			const reader = ReaderFactory.factory(source.type);
```

Comment:
This await was superfluous, since the factory doesnt return a promise

Reply:
Small relic of a time when it still was... 😅